app/views/admin/pending_orphan_lists/valid_list.html.arb

```diff
@@ -4,7 +4,7 @@ div do
   table_for result do
     column :name
     column :father_name
-    column :date_of_birth
+    column(:date_of_birth) { |orphan| format_date(Date.parse(orphan.date_of_birth)) }
```

Comment:
Instead of conforming to Rails' new default date format, convert orphans' DOBs to the common full-date format as specified by the client.